dashboard/client/src/common/CollapsibleSection.tsx

```diff
@@ -11,24 +9,27 @@ import React, {
 import { RiArrowDownSLine, RiArrowRightSLine } from "react-icons/ri";
 import { ClassNameProps } from "./props";
 
-const useStyles = makeStyles((theme) =>
-  createStyles({
-    title: {
-      display: "flex",
-      flexDirection: "row",
-      flexWrap: "nowrap",
-      alignItems: "center",
-      fontWeight: 500,
-      cursor: "pointer",
-      marginRight: theme.spacing(1),
-    },
-    icon: {
-      marginRight: theme.spacing(1),
-      width: 24,
-      height: 24,
-    },
-  }),
-);
+const TitleTypography = styled(Typography)(({ theme }) => ({
```

Comment:
for something like Typography which accepts sx, can we use `sx` props instead of styled?

Reply:
No problem, we can adjust to using `sx` as the default.

Reply:
Have replaced them with `sx`, please take a look.